packages/fuels/src/cli/commands/deploy/index.ts

```diff
@@ -13,9 +16,12 @@ import { deployContract } from './deployContract';
 import { getDeployConfig } from './getDeployConfig';
 import { saveContractIds } from './saveContractIds';
 
-export async function deploy(config: FuelsConfig) {
+export async function deploy(config: FuelsConfig, _program: Command) {
   const contracts: DeployedContract[] = [];
 
+  // always release for deployments
+  const mode: BuildMode = BuildMode.RELEASE;
```

Comment:
Does running `npx fuels build` and then `npx fuels deploy` break because `fuels build` builds in debug mode by default?

Reply:
Yes, good catch! It _would_ break, but I'll take a [different approach](https://github.com/FuelLabs/fuels-ts/pull/1784#issuecomment-1956680773) that will solve this indirectly.